slammer.go

```diff
@@ -26,6 +27,7 @@ type config struct {
 	pauseInterval time.Duration
 	workers       int
 	debugMode     bool
+	loops         int
```

Comment:
I can only interpret what you were doing with loops by the code, so tell me if i've got this right:

`loops` is a flag to specify how many times each row should be inserted, right?

So if loops = 10, and you have 10,000 rows to insert, the net effect is 100,000 rows?

I'm kinda torn on this... It's not really something I had in mind when building the library but, it's not like it's harmful when you ignore it at a default of 1...

Let me mull it over

Reply:
Yes, basically 'loops' says how many times to repeat the load. The original behavior is not changed because the default value is "1", but then one can set XYZ to stress the system iteratively without generating huge files...